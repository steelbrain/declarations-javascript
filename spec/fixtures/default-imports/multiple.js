import x from 'file-x'
import y from 'file-y'
x()
y()
