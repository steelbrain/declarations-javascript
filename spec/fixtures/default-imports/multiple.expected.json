[
  {
    "name": "x",
    "position": {
      "start": {
        "line": 3,
        "column": 0
      },
      "end": {
        "line": 3,
        "column": 1
      }
    },
    "source": {
      "name": null,
      "filePath": "file-x",
      "position": {
        "start": {
          "line": 1,
          "column": 7
        },
        "end": {
          "line": 1,
          "column": 8
        }
      }
    }
  },
  {
    "name": "y",
    "position": {
      "start": {
        "line": 4,
        "column": 0
      },
      "end": {
        "line": 4,
        "column": 1
      }
    },
    "source": {
      "name": null,
      "filePath": "file-y",
      "position": {
        "start": {
          "line": 2,
          "column": 7
        },
        "end": {
          "line": 2,
          "column": 8
        }
      }
    }
  }
]